SPECS/virtiofsd/virtiofsd.spec

```
@@ -50,8 +51,10 @@ Provides:       vhostuser-backend(fs)
 vhost-user virtio-fs device backend written in Rust
 
 %prep
-%autosetup -n %{name}-v%{version}
+%autosetup -n %{name}-v%{version} -N
 tar -xf %{SOURCE1}
+ls -l
+%patch -P 0 -p1
```

consider using `%autopatch` here
```suggestion
%autopatch -p1
```